src/bindings/pygaia/scripts/classification/generate_classification_project.py

```diff
@@ -23,13 +23,22 @@
 from __future__ import with_statement
 import sys
 from os.path import join, abspath
-from optparse import OptionParser
+from argparse import ArgumentParser
 from gaia2 import filedir
 import gaia2.fastyaml as yaml
 
-PROJECT_TEMPLATE = open(join(filedir(), 'classification_project_template.yaml')).read()
 
-def generateProject(groundtruth_file, filelist_file, project_file, datasets_dir, results_dir, seed=None, cluster_mode=False):
+# Use a dict to map essentia versions to the appropriate template so that we
+# don't need to store one template for each version.
+VERSION_MAP = {
+    '2.1-beta5': '2.1-beta5',
+    '2.1-beta6': '2.1-beta5',
+}
+
+
+def generateProject(groundtruth_file, filelist_file, project_file, datasets_dir,
```

Comment:
I've been slowly updating methods here to use _ instead of capital letters - `generate_project` instead of `generateProject`